test/adaptive_load/scoring_function_test.cc

```diff
@@ -0,0 +1,180 @@
+#include "external/envoy/source/common/config/utility.h"
+
+#include "adaptive_load/scoring_function_impl.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+
+TEST(BinaryScoringFunctionConfigFactoryTest, GeneratesEmptyConfigProto) {
+  auto& config_factory =
+      Envoy::Config::Utility::getAndCheckFactoryByName<ScoringFunctionConfigFactory>(
+          "nighthawk.binary");
+  const Envoy::ProtobufTypes::MessagePtr message = config_factory.createEmptyConfigProto();
```

Comment:
message seem like a generic variable name, can we name this a more specific name like empty_config

Reply:
Done